### PR TITLE
Change log v51.8.0

### PR DESCRIPTION
**Pull request by AnanyaJha**
### What does this PR do?
Changelog for release v51.8.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
+# 51.8.0 - April 07, YYYY
```
Comment (ekapner) [inline packages/salesforcedx-vscode/CHANGELOG.md:1]:
```suggestion
# 51.8.0 - April 07, 2021
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+
+- Add cancellation to SObject Refresh command ([PR #3116](https://github.com/forcedotcom/salesforcedx-vscode/pull/3116))
+
+## Fixed
+
+#### docs
+
+- Update Org Picker location in documentation ([PR #3099](https://github.com/forcedotcom/salesforcedx-vscode/pull/3099)) ([Issue #3095](https://github.com/forcedotcom/salesforcedx-vscode/issues/3095))
+
+- Update Apex status to reflect default usage in CLI ([PR #3119](https://github.com/forcedotcom/salesforcedx-vscode/pull/3119))
```
Comment (ekapner) [inline packages/salesforcedx-vscode/CHANGELOG.md:19]:
```suggestion
- Update Apex status to reflect default usage in Salesforce CLI ([PR #3119](https://github.com/forcedotcom/salesforcedx-vscode/pull/3119))
```

**Review by ekapner — APPROVED** [review]
A couple minor changes

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 2 above)
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:19]: @AnanyaJha are you porting this PR to the release branch ? I don't see it there yet, only in `develop`.

Comment (AnanyaJha) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Yessir port PR is here https://github.com/forcedotcom/salesforcedx-vscode/pull/3130

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Awesome 👍

**Review by lcampos — APPROVED** [review]

**Review by xyc — APPROVED** [review]

**Pull request merged**